main.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+import tdl
+
+from render import render_all, clear_all
+from colors import colors
+from input import handle_key_input
+from entities import Character
+from dungeon import Dungeon
+
+
+def main():
+    # Constants
+    # A lot of these are just hardcoded values for debugging purposes. Ideally, they would
+    # be changeable by in-game settings.
+    SCREEN_WIDTH = 80
+    SCREEN_HEIGHT = 50
+    MAP_WIDTH = 80
+    MAP_HEIGHT = 44
+    GAME_TITLE = "Tonzo Studios Roguelike"
+    MAX_ROOMS = 30
+    MIN_ROOM_SIZE = 6
+    MAX_ROOM_SIZE = 10
+    MAX_ENTITIES_PER_ROOM = 3
+    FOV_ALGORITHM = "BASIC"
+    FOV_LIGHT_WALLS = True
+    FOV_RADIUS = 10
+
+    # Initialize console
+    tdl.set_font('lucida10x10_gs_tc.png', greyscale=True, altLayout=True)
+    console = tdl.Console(SCREEN_WIDTH, SCREEN_HEIGHT)
+    root_console = tdl.init(SCREEN_WIDTH, SCREEN_HEIGHT, title=GAME_TITLE, fullscreen=False)
+
+    # Initialize player entity
+    # FIXME: Character is an abstract class
```

Comment:
I'll do the composition thing whenever I get a chance and scrap the player PR or rather, merge it into the character class